types/node/buffer.d.ts

```diff
@@ -2,6 +2,8 @@ declare module "buffer" {
     export const INSPECT_MAX_BYTES: number;
     const BuffType: typeof Buffer;
 
+    export function transcode(source: Buffer | Uint8Array, fromEnc: string, toEnc: string): Buffer;
```

Comment:
You could consider to use a `"ascii" | "utf8" | "utf16le" | "ucs2" | "latin1" | "binary"` instead `string`. I know something similar would be applicable at several other places but changing that will be a breaking change but this is change here is new (see also #33043)

Reply:
Good idea. Changed.